tools/topology/topology2/include/components/gain.conf

```diff
@@ -92,6 +92,15 @@ Class.Widget."gain" {
 			"index"
 			"instance"
 		]
+		#
+		# mandatory attributes that must be provided when the buffer class is instantiated
+		#
+		!mandatory [
+			"no_pm"
+			"uuid"
+			"num_sink_pins"
+			"num_source_pins"
+		]
```

Comment:
I am struggling to see what the no_pm and uuid have to do with the rest, and what that buffer class comment is about.

Reply:
I will remove this spurious change.